splunk_add_on_ucc_framework/ucc_ui_lib/src/main/webapp/components/table/CustomTable.jsx

```diff
@@ -127,171 +63,54 @@ function CustomTable({
-                                                statusMapping
-                                                    ? statusMapping[0].mapping.false
-                                                    : 'Enabled'
-                                            )}
-                                            unselectedLabel={_(
-                                                statusMapping
-                                                    ? statusMapping[0].mapping.true
-                                                    : 'Disabled'
-                                            )}
-                                        />
-                                        {statusContent}
-                                    </div>
-                                </Table.Cell>
-                            );
-                        } else if (header.field === 'actions') {
-                            cellHTML = rowActionsPrimaryButton(row);
-                        } else {
-                            cellHTML = (
-                                <Table.Cell key={header.field}>{row[header.field]}</Table.Cell>
-                            );
-                        }
-                        return cellHTML;
-                    })}
-            </Table.Row>
-        );
-    };
+    }, [columns, handleSort, sortDir, sortKey]);
 
     const getTableBody = () => {
         return (
-            <Table.Body>{data && data.length && data.map((row) => getTableRow(row))}</Table.Body>
+            <Table.Body>
+                {data &&
+                    data.length &&
+                    data.map((row) => {
+                        return (
+                            <CustomTableRow
+                                key={row.id}
+                                row={row}
+                                columns={columns}
+                                statusMapping={statusMapping}
+                                unifiedConfigs={unifiedConfigs}
+                                page={page}
+                                handleToggleActionClick={handleToggleActionClick}
+                                {...(moreInfo
+                                    ? {
+                                          expansionRow: getExpansionRow(
+                                              columns.length,
+                                              row,
+                                              moreInfo
+                                          ),
+                                      }
+                                    : {})}
+                            />
+                        );
+                    })}
+            </Table.Body>
         );
     };
 
     return (
         <>
             {columns && columns.length && (
-                <Table stripeRows {...(moreInfo ? { rowExpansion: 'single' } : {})}>
-                    {getTableHeaders()}
+                <Table id="test" stripeRows {...(moreInfo ? { rowExpansion: 'single' } : {})}>
```

Comment:
@tbalar-splunk Can we remove `id="test"` if it is not required?

Reply:
Ohh sorry 
It was by mistake